_pages/production.md

```diff
@@ -25,11 +25,11 @@ sidenav:
 
 Our integration documentation includes endpoint urls for our sandbox environment <https://idp.int.identitysandbox.gov/>. Our production environment is located at <https://secure.login.gov/>. The URL path to each endpoint remains the same. Only the domain will change. For example, the authorization endpoint will change as follows:
 * OpenID Connect: <https://secure.login.gov/openid_connect/authorize>
-* SAML: <https://secure.login.gov/api/saml/auth2021>
+* SAML: <https://secure.login.gov/api/saml/auth2022>
```

Comment:
should we make a variable for these? We could do a page variable for `saml_year` or we could just do it based on the currenet calendar year?

```suggestion
* SAML: <https://secure.login.gov/api/saml/auth{{ page.saml_year }}>
```

```suggestion
* SAML: <https://secure.login.gov/api/saml/auth{{ site.time | date: "%Y" }}>
```

Reply:
(and the rest of the endpoints too)

Reply:
That would be ace! I can see if I have some bandwidth tonight to make that happen.